### PR TITLE
update game screen UI

### PR DESCRIPTION
**Pull request by segunolalive**
#### What does this PR do?
Cleans up game screen UI

#### Description of Task to be completed?
Add extra styling to game screen 

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#151754992

#### Screenshots (if appropriate)
localhost-3000-

#### Questions:

### CODE REVIEW

**Review by nebanat — APPROVED** [review]
Good job

**Review by ibonly — APPROVED** [review]

File: public/js/controllers/game.js [hunk 1]
```diff
@@ -264,7 +264,11 @@ angular.module('mean.system')
 
       $scope.abandonGame = function () {
         game.leaveGame();
```
Comment (gbengaoyetade) [inline public/js/controllers/game.js:266]: I'll still love it if we can achieve the alert before leaving game feature

**Review by gbengaoyetade — APPROVED** [review]
Nice work on the UI

File: public/views/app.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
             <span></span>
           </a>
           <ul class="nav-list right hide-on-med-and-down">
-            <li class="nav-item"><a class="active" href="/#!">Home</a></li>
+            <li class="nav-item"><a class="" href="/#!">Home</a></li>
```
Comment (dinobi) [inline public/views/app.html:15]: remove unused css class

**Review by enodi — APPROVED** [review]

**Pull request merged**